OneSignalSDK/onesignal/src/main/java/com/onesignal/ADMMessageHandlerJob.kt

```diff
@@ -0,0 +1,46 @@
+package com.onesignal
+
+
+import android.content.Context
+import android.content.Intent
+import com.amazon.device.messaging.ADMMessageHandlerJobBase
+
+class ADMMessageHandlerJob : ADMMessageHandlerJobBase() {
+
+    override fun onMessage(context: Context?, intent: Intent?) {
+        val bundle = intent!!.extras
```

Comment:
we shouldn't use force unwrap, always try to treat optional with null safety methods